code/modules/surgery/organs_internal.dm

```diff
@@ -98,6 +98,71 @@
 			if(I && I.damage > 0)
 				I.take_damage(dam_amt,0)
 
+/datum/surgery_step/internal/fix_organ_rob
+	allowed_tools = list(
+	/obj/item/stack/nanopaste = 100
+	)
+
+	min_duration = 70
+	max_duration = 90
+
+	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
```

Comment:
Do not use relative pathing. Procs should be `/datum/surgery_step/internal/fix_organ_rob/can_use(` and such.